google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -3547,4 +3547,36 @@ HmacKeyMetadata updateHmacKeyState(
    * @throws StorageException upon failure
    */
   ServiceAccount getServiceAccount(String projectId);
+
+  /**
+   * Creates a notification with the specified entity on the specified bucket.
+   *
+   * @return the notification that was created.
+   * @throws StorageException upon failure
+   */
+  Notification createNotification(String bucket, NotificationInfo notificationInfo);
```

Comment:
Does `creates a notification` mean subscription?

Reply:
yes

Reply:
It should be reflected in the spec.